packages/manager/apps/public-cloud/src/assets/theme/dark/index.less

```diff
@@ -367,11 +367,11 @@
   }
 
   .oui-datagrid__row:nth-child(odd) .oui-datagrid__cell {
-    background-color: #181818;
+    background-color: #181818 !important;
   }
 
   .oui-datagrid__row:nth-child(even) .oui-datagrid__cell {
-    background-color: #1d1d1d;
+    background-color: #1d1d1d !important;
```

Comment:
What's overriding the style ? Maybe the selector should be more specific

Reply:
I'm facing the following selector:

```css
.oui-datagrid .oui-datagrid__cell {
  background-color: #fff !important;
  ...
}
```

Which is causing me troubles. I cannot be more specific with a selector,
so that's why I had to add a `!important`.

Reply:
Ok for now (due to https://github.com/ovh/ovh-ui-kit/blob/v2.x/packages/ovhcloud-light-theme/components/_datagrid.less#L9